src/pages/App.js

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from 'react';
```

Comment:
Are the changes in `App.js` and `Profile.js` meant to be included in this PR? It seems like they're the same changes in #38, so I think they should be excluded from this PR.

Reply:
Good call I will remove these from this PR to have more focus on testing my mistake